nixos/modules/services/hardware/fancontrol.nix

```diff
@@ -4,42 +4,41 @@ with lib;
 
 let
   cfg = config.hardware.fancontrol;
-  configFile = pkgs.writeText "fan.conf" cfg.config;
-
-in {
+  configFile = pkgs.writeText "fancontrol" cfg.config;
```

Comment:
Why not naming the file "fancontrol.conf"?

Reply:
"fancontrol" is the default config file name generated by pwmconfig
it also seems like that name is not particularly relevant?

Reply:
The name does not matter for functionality but I think it would increase readability.